lib/model/model.go

```diff
@@ -809,6 +809,16 @@ func (m *Model) ClusterConfig(deviceID protocol.DeviceID, cm protocol.ClusterCon
 			continue
 		}
 
+		if m.cfg.IgnoredFolder(folder.ID) {
+			for _, dev := range folder.Devices {
+				// Find friendly name instead of dev id and report that to console
```

Comment:
Sorry, one last nitpick :/ We always log the device ID to console. (We should, at some point, gain a deviceDescription method like we have for folders, to print a short ID and label, but we don't currently have that.)